sdks/python/apache_beam/io/gcp/pubsub_test.py

```diff
@@ -630,13 +620,13 @@ def test_write_messages_unsupported_features(self, mock_pubsub):
       p.run()
     options = PipelineOptions([])
     options.view_as(StandardOptions).streaming = True
-    p = TestPipeline(options=options)
-    _ = (p
-         | Create(payloads)
-         | WriteToPubSub('projects/fakeprj/topics/a_topic',
-                         timestamp_attribute='timestamp'))
     with self.assertRaisesRegex(NotImplementedError,
                                 r'timestamp_attribute is not supported'):
+      p = TestPipeline(options=options)
```

Comment:
There's no `with` added here. (Probably these two separate pipelines should be in two separate tests to begin with.)